activesupport/test/core_ext/duration_test.rb

```diff
@@ -71,6 +71,9 @@ def test_inspect
     assert_equal "7 days",                          7.days.inspect
     assert_equal "1 week",                          1.week.inspect
     assert_equal "2 weeks",                         1.fortnight.inspect
+    assert_equal "0 seconds",                       ActiveSupport::Duration.new(0, []).inspect
```

Comment:
`ActiveSupport::Duration.new` isn't part of the public API, so we don't need to add tests for it; the one below using `%` is sufficient.

Reply:
done